components/uuid/uuid.c

```diff
@@ -0,0 +1,131 @@
+#include "uuid.h"
```

Comment:
If, I assume you copied and pasted this snippet, can we had a link to the exact hash on GitHub for the project? So we have a direct reference to the license at this point in time.

Reply:
Good idea.